style/pi-hole.css

```diff
@@ -1609,3 +1612,13 @@ textarea.field-sizing-content {
   opacity: 0;
   transition: opacity 200ms ease-in-out;
 }
+
+#StaticDHCPTable td.table-danger {
+  background-color: #ff000022 !important;
+  color: #fff !important;
+}
+
+#StaticDHCPTable td.table-success {
+  background-color: #74c70022 !important;
+  color: #fff !important;
+}
```

Comment:
Quick suggestion just to improve the colors:

```suggestion
#StaticDHCPTable .table-danger {
  background-color: #d337 !important;
}

#StaticDHCPTable .table-success {
  background-color: #2c24 !important;
}
```

Still using semi-transparent colors, but now the contrast is good enough when using the original text color, so I think we don't need to change `color:`.

Reply:
Oh, I did not see this before. I pushed non-transparent suggestions that should be the same on all variants, otherwise. Not overly pretty but it may suffice. Otherwise, we'll theme-dependent colors and this PR already became a lot more involved than I had hoped initially.

<img width="1014" height="205" alt="image" src="https://github.com/user-attachments/assets/0d3aec37-d10e-4f10-b1fd-12b4155a96c8" />